doc/monitoring/api_reference.rst

```diff
@@ -46,7 +46,8 @@ Counter
 
         :param number        num: Increase value.
         :param table label_pairs: Table containing label names as keys,
-                                  label values as values.
+                                  label values as values. Note that all
+                                  numbers in label_pairs are treated as strings.
```

Comment:
```suggestion
                                  keys in label_pairs are treated as strings.
```

Reply:
I think it's closer to the truth

Reply:
Maybe 'label values'?
Keys sound like "label names"

Reply:
Isn't it both keys and values in table pairs? They both concatenated on metric observation as strings and casted to strings in both json and Prometheus export

Reply:
Maybe "Note that **both labels names and values** in label_pairs are treated as strings" ?

Reply:
Seems legit